httpcache.go

```diff
@@ -350,9 +359,64 @@ type ctxKey string
 
 const getterContextCtxKey ctxKey = "getter_context"
 
+// UnmarshalCaddyfile sets up the handler from Caddyfile tokens. Syntax:
+//
+//     cache {
+//         default_ttl <ttl>
+//         max_size <size>
+//         olric_env <env>
```

Comment:
`olric_env foo`

or 

```
olric {
   env foo
}
```

What's the usual convention @mholt @francislavoie?

Reply:
Are there other `olric` options to go in? If so, grouping it might make sense, but otherwise, why not just `env`?

Depends if we want to "leak" that olric is the underlying implementation. Does that matter to the user? We'd need to explain what "olric" is for them to know what that's intended for.

Reply:
I think there will be more to support the clustered mode (cc @buraksezer).

Reply:
My impulse is to prefer `env foo` as a subdirective. :+1:

Reply:
Olric has many configuration directives. Mostly key/value but it can also contain lists and maps. I don't know how we translate it into Caddyfile format. Here is a sample for `olricd` (includes service discovery sections for Consul): https://github.com/buraksezer/olric/blob/master/docker/olricd-consul.yaml

If it's complicated to build or maintain, we may read the yaml file from the disk and translate into the configuration for embedded mode. `olricd` is already doing this job. 

```
cache {
    default_ttl 5
    olric_config /path/to/olric.yaml
}
```

Reply:
@buraksezer do you think that it could be possible to move the YAML parsing logic (https://github.com/buraksezer/olric/blob/master/cmd/olricd/server/config.go#L32) directly in `github.com/buraksezer/olric/config` (or in a dedicated package)? So we can reuse the logic of Olricd directly, without copying it (and then having to maintain it) it inside this module?

Reply:
@dunglas I'll do it and release a new beta in a couple of days.

Reply:
Thanks!!

Reply:
@dunglas `config.Load` function is ready to use. See [v0.3.0-beta.6](https://github.com/buraksezer/olric/releases/tag/v0.3.0-beta.6)

Simply:
```go
c, err := config.Load(cpath)
if err != nil {
// handle this error
}
```

Reply:
(btw @buraksezer I think you have the first two issues in that changelog flipped)

Reply:
@francislavoie Thanks. Fixed!

Reply:
Support added in 9a0d86c!